File: almanac/util/file.lua
local json = require("almanac.lib.json")

local module = {}

function module.read(path)
    local file = io.open(path, "r")
    
    if not file then
        error("File not found! " .. path)
    end
    
    local text = file:read("*a")
    file:close()
    
    return text
end

function module.json_read(path)
    local function thing()
        return json.decode(module.read(path))
    end
    
    local status, data = pcall(thing)
    
    if status then
        return data
        
    else
        error("JSON ERROR: " .. path .. "|" .. data)
    end
end

function module.exists(path)
    local file = io.open(path, "r")
    
    return file ~= nil and file:close()
end

return module

File: oifey/util/file.py
import json, os

def json_read(path):
    with open(path, "r", encoding="utf-8") as f:
        return json.loads(f.read())
        
def json_write(path, data):
    with open(path, "w", encoding="utf-8") as f:
        return f.write(json.dumps(data, indent=4))
        
def create_folder(path):
    if not os.path.exists(path):
        os.makedirs(path)